### PR TITLE
initial release of pam interactive auth plugin

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/initial-release-of-the-irods-pam-interactive-auth-plugin.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Status: published
+
+
+The iRODS Consortium is pleased to announce the first release of the [PAM Interactive Authentication Plugin](https://github.com/irods/irods_auth_plugin_pam_interactive).
+
+It is released today as v0.1.0.
+
+The plugin enables organizations to develop dynamic authentication flows that meet their needs. Using the plugin, we've demonstrated working authentication flows for OpenID Connect and Kerberos. Configuration for various authentication flows is documented in the README.
+
+The initial implementation was developed by SURF and completed by the iRODS Consortium.
```
Comment (trel) [inline content/posts/initial-release-of-the-irods-pam-interactive-auth-plugin.md:14]:
```suggestion
The initial implementation was developed by [SURF](https://www.surf.nl/en) and completed by the iRODS Consortium.
```

**Review by alanking — COMMENTED** [review]
New wording looks good

File: content/posts/initial-release-of-the-irods-pam-interactive-auth-plugin.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Watch the [TRiRODS talk](https://youtu.be/heFIGvlHhck?si=lYsk3XXBXBmG4_uk) or visit the [GitHub repository](https://github.com/irods/irods_auth_plugin_pam_interactive/tree/0.1.0) to learn more.
```
[the remark this answers is not in the document]
Comment (korydraughn) [reply content/posts/initial-release-of-the-irods-pam-interactive-auth-plugin.md]: Done.

File: content/posts/initial-release-of-the-irods-pam-interactive-auth-plugin.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/heFIGvlHhck?si=CP6-SG9PqeO33HGI" frameborder="0" gesture="media" allow="encrypted-media" allowfullscreen></iframe>
```
[the remark this answers is not in the document]
Comment (korydraughn) [reply content/posts/initial-release-of-the-irods-pam-interactive-auth-plugin.md]: Done.

File: content/posts/initial-release-of-the-irods-pam-interactive-auth-plugin.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/initial-release-of-the-irods-pam-interactive-auth-plugin.md]: Done.

**Comment by korydraughn** [comment]
Review comments resolved.

**Comment by trel** [comment]
squash it

**Comment by korydraughn** [comment]
Squashed.

**Comment by trel** [comment]
mergin'

**Pull request merged**